.github/workflows/codeql-analysis.yml

```diff
@@ -3,13 +3,9 @@ name: "CodeQL"
 on:
   push:
     branches: ["master", "[0-9].[0-9]"]
-    paths:
```

Comment:
Looks like CodeQL is setup to work on both python and javascript, so let's trigger at every run. This is a short/cheap step AFAIK, so no need to limit.